charts/sftp-server/templates/service.yaml

```diff
@@ -6,6 +6,11 @@ metadata:
     {{- include "sftp-server.labels" . | nindent 4 }}
 spec:
   type: {{ .Values.service.type }}
+  {{- if $.Values.service.externalTraffic.enabled }}
+  {{- if eq "ClusterIP" $.Values.service.serviceType }}
```

Comment:
Is the `externalTrafficPolicy` limited to the `ClusterIP` service? I didn't see this.

https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip

Reply:
Indeed you're right, i deleted that if